scripts/howey.py

```diff
@@ -0,0 +1,96 @@
+"""Parse tIVT .npy data from the Howey dataset
```

Comment:
Could you group these two scripts in a single folder, then add a README to explain what they are for?